proc_scripts/load_data.py

```diff
@@ -76,13 +76,11 @@ def proc_Hahn_echoph(s, fl=None):
     SW_kHz = s.get_prop('acq_params')['SW_kHz']
     nScans = s.get_prop('acq_params')['nScans']
     s.reorder('t',first=True)
-    t2_axis = s.getaxis('t')[0:2048]
     s.setaxis('t',None)
     s.chunk('t',['ph2','ph1','t2'],[2,4,-1])
     s.labels({'ph2':r_[0.,2.]/4,
         'ph1':r_[0.,1.,2.,3.]/4})
     s.reorder(['ph2','ph1'])
-    s.setaxis('t2',t2_axis)
```

Comment:
I removed this b/c I didn't see a need to mess with the time axis

Reply:
We need these so that when we call 't2' it calls the original 't' axis prior to being chunked and has all the points. Without these lines:
`ValueError: seems to be no axis for't2'set an axis before you try to FT`

Reply:
took these lines out as well as s.setaxis('t',None) and it runs fine prior to master being merged